projects/1.19/assets/naturalist/naturalist/lang/zh_cn.json

```
@@ -0,0 +1,226 @@
+  "item.naturalist.tortoise_spawn_egg": "乌龟刷怪蛋",
+  "item.naturalist.duck_spawn_egg": "鸭刷怪蛋",
+
+
+  "item.naturalist.antler": "鹿角",
+  "item.naturalist.snail_bucket": "蜗牛桶",
+  "block.naturalist.chrysalis": "蛹",
+  "block.naturalist.teddy_bear": "玩具熊",
+  "block.naturalist.alligator_egg": "短吻鳄蛋",
+  "block.naturalist.azure_froglass": "蔚蓝蛙玻璃",
+  "block.naturalist.verdant_froglass": "青翠蛙玻璃",
+  "block.naturalist.crimson_froglass": "绯红蛙玻璃",
+  "item.naturalist.cooked_venison": "熟鹿肉",
+  "item.naturalist.bass": "生鲈鱼",
+  "item.naturalist.cooked_bass": "熟鲈鱼",
+  "item.naturalist.glow_goop": "荧光黏胶",
+  "item.naturalist.venison": "生鹿肉",
+  "item.naturalist.bear_fur": "熊皮",
+  "item.naturalist.catfish": "生鲶鱼",
+  "item.naturalist.cooked_catfish": "熟鲶鱼",
+  "item.naturalist.catfish_bucket": "鲶鱼桶",
+  "item.naturalist.fish_fillet": "生鱼排",
+  "item.naturalist.cooked_fish_fillet": "熟鱼排",
+  "item.naturalist.bass_bucket": "鲈鱼桶",
+  "item.naturalist.snail_shell": "蜗牛壳",
+  "item.naturalist.lizard_tail": "蜥蜴尾",
+  "item.naturalist.cooked_lizard_tail": "熟蜥蜴尾",
+  "block.naturalist.tortoise_egg": "乌龟蛋",
+  "item.naturalist.worm": "蚯蚓",
+
+  "item.naturalist.duck": "生鸭肉",
+  "item.naturalist.cooked_duck": "熟鸭肉",
+  "item.naturalist.duck_egg": "鸭蛋",
+  "item.naturalist.cooked_egg": "煎蛋",
+  "item.naturalist.reptile_hide": "爬行动物皮",
+
+  "block.naturalist.cattail": "香蒲",
+  "item.naturalist.cattail_fluff": "香蒲绒毛",
+  "block.naturalist.duckweed": "浮萍",
+
+  "block.naturalist.limestone": "石灰岩",
+  "block.naturalist.limestone_bricks": "石灰岩砖",
+  "block.naturalist.cut_limestone": "切制石灰岩",
+  "block.naturalist.smooth_limestone": "平滑石灰岩",
+
+
+
+  "item.minecraft.tipped_arrow.effect.forest_dasher": "森林疾行之箭",
+  "item.minecraft.potion.effect.forest_dasher": "森林疾行药水",
+  "item.minecraft.splash_potion.effect.forest_dasher": "喷溅型森林疾行药水",
+  "item.minecraft.lingering_potion.effect.forest_dasher": "滞留型森林疾行药水",
+  "item.minecraft.tipped_arrow.effect.glowing": "荧光之箭",
+  "item.minecraft.potion.effect.glowing": "荧光药水",
+  "item.minecraft.splash_potion.effect.glowing": "喷溅型荧光药水",
+  "item.minecraft.lingering_potion.effect.glowing": "滞留型荧光药水",
+
+
+  "naturalist.subtitles.entity.bird.hurt": "鸟：受伤",
+  "naturalist.subtitles.entity.bird.death": "鸟：死亡",
+  "naturalist.subtitles.entity.bird.eat": "鸟：进食",
+  "naturalist.subtitles.entity.bird.fly": "鸟：扑翼",
+  "naturalist.subtitles.entity.bird.ambient_bluejay": "冠蓝鸦：鸣叫",
+  "naturalist.subtitles.entity.bird.ambient_canary": "金丝雀：鸣叫",
+  "naturalist.subtitles.entity.bird.ambient_cardinal": "主红雀：鸣叫",
+  "naturalist.subtitles.entity.bird.ambient_robin": "旅鸫：鸣叫",
+  "naturalist.subtitles.entity.deer.hurt": "鹿：受伤",
+  "naturalist.subtitles.entity.deer.ambient": "鹿：呼噜",
+  "naturalist.subtitles.entity.deer.hurt_baby": "幼鹿：受伤",
+  "naturalist.subtitles.entity.deer.ambient_baby": "幼鹿：吼叫",
+  "naturalist.subtitles.entity.elephant.hurt": "大象：受伤",
+  "naturalist.subtitles.entity.elephant.ambient": "大象：吼叫",

```

```suggestion
  "naturalist.subtitles.entity.deer.ambient_baby": "幼鹿：鸣叫",
  "naturalist.subtitles.entity.elephant.hurt": "大象：受伤",
  "naturalist.subtitles.entity.elephant.ambient": "大象：鸣鼻",
```
象鸣/鹿鸣？
这俩动物的"吼叫"是啥声（）

---

这个好